performance/README.md

```diff
@@ -0,0 +1,114 @@
+# Howso Platform Performance Considerations
+
+## Overview
+
+This guide covers performance considerations for the Howso Platform and client installation. 
+
+## TLDR
+- Use a dedicated Kubernetes cluster for Howso Platform.
+- Increase the size of NATS, Redis, Postgres.
+- Use cluster autoscaling.
+- Run workloads from a large dedicated client machine.
+
+## Cluster Configuration
+
+### Dedicated Cluster
+
+In the Howso Platform, trainees (also known as workers) are created as new (stateful set) pods on demand.  
+- Individual trainees can be very large - it is not uncommon to dedicate tens of cpu resources and even more Gb of memory to a single trainee.
+- The Kubernetes API server is used to create and manage these pods.  The API server is a critical component of the cluster, can come under load, potentially impacting other cluster workloads. 
```

Comment:
Is there a specific recommendation here, beyond having more than one API server?

Reply:
Yeah - I have plans for a later update with more on this.